### ISSUE TITLE
normalize ='minabs' not working for ParallelCoordinates

### ISSUE THREAD

**Issue by haleemason** [DistrictDataLabs/yellowbrick#797]
**Describe the bug**
When using `visualizer = ParallelCoordinates(normalize ='minabs')` the normalize parameter  ='minabs' is not working.

**To Reproduce**

```python
from yellowbrick.features import ParallelCoordinates

# Load data set (don't need to do again, included for completeness)
X, y = load_occupancy()

# Specify the features of interest and the classes of the target
features = [
    "temperature", "relative humidity", "light", "CO2", "humidity"
]
classes = ["unoccupied", "occupied"]

# Instantiate the visualizer
visualizer = ParallelCoordinates(
    classes=classes, features=features,
    normalize='minabs', # normalize options: minmax, minabs, standard, l1, or l2
    sample=0.05, shuffle=True, size=(1080, 720)
)

# Fit the visualizer and display it
visualizer.fit_transform(X, y)
visualizer.poof()
```

**Dataset**
Did you use a specific dataset to produce the bug? Where can we access it?
`python -m yellowbrick.download`
`X, y = load_occupancy()`

**Expected behavior**
I was expecting the visualizer plot to be normalized using the `minabs` method. 

**Traceback**

```
---------------------------------------------------------------------------
YellowbrickValueError                     Traceback (most recent call last)
<ipython-input-21-c85f2a8aee8c> in <module>()
      3     classes=classes, features=features,
      4     normalize='minabs', # This time we'll specify a normalizer
----> 5     sample=0.05, shuffle=True, size=(1080, 720)
      6 )
      7 

~/repos/dev/yellowbrick/yellowbrick/features/pcoords.py in __init__(self, ax, features, classes, normalize, sample, random_state, shuffle, color, colormap, alpha, fast, vlines, vlines_kwds, **kwargs)
    260             raise YellowbrickValueError(
    261                 "'{}' is an unrecognized normalization method"
--> 262                 .format(normalize)
    263             )
    264 

YellowbrickValueError: 'minabs' is an unrecognized normalization method
<Figure size 1080x720 with 0 Axes>
```

**Desktop (please complete the following information):**
 - OS: `macOS`
 - Python Version:  `[Python 3.6.5 |Anaconda, Inc.| (default, Apr 26 2018, 08:42:37)`
`[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin`
 - Yellowbrick Version: `yellowbrick-0.9.1`

**Comment by pswaldia** [comment]
Yellowbrick ParallelCoordinates do not have 'minabs' as an option for normalization. Mostly used Normalizing methods are 'min-max', 'maxabs' , 'l1' , 'l2' in practice.

P.S  MinAbsScaler() is not implemented in sklearn.preprocessing module.

**Comment by haleemason** [comment]
Okay, cool. The documentation stated, "Several transformers are available; try using minmax, minabs, standard, l1, or l2 normalization to change perspectives in the parallel coordinates". I've updated the [documentation ](https://www.scikit-yb.org/en/latest/api/features/pcoords.html) to remove the recommendation to try `minabs` for ParallelCoordinates. I have submitted a PR for this "bug". Thanks for your quick feedback!
